src/FormField.js

```diff
@@ -108,8 +113,8 @@ class InputField extends React.Component {
     if (label) {
       LabelChild = <Label>{label}</Label>
     }
-    if (!InputChild) {
-      InputChild = <Input />
+    if (!FieldChild) {
+      FieldChild = <Input />
     }
```

Comment:
Non-blocking, outside the scope of this PR, though the observation is about these `if` and inner `if` statements inside each `React.Children.forEach` iteration. Perhaps a new issue can be log to start a discuss about potential pattern to "_dry_" out all these `if` & inner `if` statements? 😇

Reply:
Similar to my comment above just preserving the existing logic to keep things small but a follow up would be nice to simplify things.